blockchains/smartchain/assets/0x7B7D3B71A12906270b1b4c187C75f45188E9aEf6/info.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "VIDTC Precious metals and stones",
+    "website": "https://about.v-id.org",
+    "short_description": "NFTs for valuables like diamonds or gold",
+    "explorer": "https://bscscan.com/address/0x7b7d3b71a12906270b1b4c187c75f45188e9aef6"
```

Comment:
This one as well to `https://bscscan.com/token/0x7b7d3b71a12906270b1b4c187c75f45188e9aef6`